public/css/brands.css

```diff
@@ -451,4 +451,10 @@ button {
 .button.button-gitbucket {
   color: #000;
   background-color: #fff
+}
+
+/* Shazaom */
```

Comment:
typo in comment iguess, shouldnt it say "Shazam"?

Reply:
> typo in comment I guess, shouldnt it say "Shazam"?

Good catch!